app/(service)/my-store/my-recruits/edit/layout.tsx

```diff
@@ -0,0 +1,14 @@
+import type { Metadata } from 'next';
+
+export const metadata: Metadata = {
+  title: 'The Julge',
+  description: 'The Julge',
+};
+
+export default function RootLayout({
+  children,
+}: Readonly<{
+  children: React.ReactNode;
```

Comment:
P3: 통일성을 위해 react에서 import 시켜주시면 좋을 것 같습니다ㅏㅏ

```suggestion
  children: ReactNode;
```